### PR TITLE
remove the anonymous user nonsense

### PR DESCRIPTION
**Pull request by ahaurw01**

### CODE REVIEW

File: app/routes/chat.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  beforeModel: function () {
```
Comment (SirZach) [inline app/routes/chat.js]: so...what do you think about letting someone watch the chat but not being able to participate unless they log in? i'm totally cool with this for now but maybe something we can add in the future? i totally dig it in twitch watching the madness that is their chat despite not being logged in

File: app/routes/deck.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  beforeModel: function () {
```
Comment (SirZach) [inline app/routes/deck.js]: another future improvement will be to not show the "Create" or "Import" buttons unless they sign in

File: app/routes/games.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  beforeModel: function () {
```
Comment (SirZach) [inline app/routes/games.js]: same that we can potentially allow people watch games without logging in but this is cool for now

File: app/utils/name-generator.js [hunk 4]
```diff
@@ -1,655 +0,0 @@
...
-  'Wombat',
-  'Woodlouse',
-  'Woodpecker',
-  'Wrasse',
-  'Yak',
-  'Zebra',
-  'Zebu',
-  'Zonkey',
-  'Zorse'
-];
```
Comment (SirZach) [inline app/utils/name-generator.js]: i'm sad in the face

Comment (ahaurw01) [inline app/utils/name-generator.js]: Maybe there's room for Prickly Puffin elsewhere in the app. :smile:

**Comment by ahaurw01** [comment]
Totally on board with allowing degraded access to those features for people not logged in. Just wanted to purge the anonymous logic for now, then can slowly add that other stuff back in.

**Comment by ahaurw01** [comment]
All good reminders. I'll create an issue marking all those guys.

**Pull request merged**